observatory-platform/observatory/platform/terraform/terraform_builder.py

```diff
@@ -31,8 +31,8 @@
 from observatory.platform.utils.proc_utils import stream_process
 
 
-def copy_dir(source_path: str, destination_path: str, ignore):
-    distutils.dir_util.copy_tree(source_path, destination_path)
+def copy_dir(source_path: str, destination_path: str, ignore, update: int = 0):
+    distutils.dir_util.copy_tree(source_path, destination_path, update=update)
```

Comment:
It looks like the `update` parameter is already set to zero by default in `distutils.dir_util.copy_tree`, so we shouldn't need to set it.

I seem to have messed up some of this code, because the ignore parameter doesn't get passed to anything. Maybe we were originally using `shutil.copytree` as that has an ignore parameter? We should probably either remove the ignore parameter or figure out if we could use shutil.copytree which can ignore files with patterns.